### DISCUSSION TITLE
Multiple parallel animations in a single config

### DISCUSSION THREAD

**Discussion by dcop** [pmndrs/react-spring#1806] [Support] [answered]
Hey everyone,

I'm struggling trying to find a method to have a bunch of parallel animations in a single config (with delay and duration for each one). 
I've tried any available hook or using the array in `to` property when defining the config, but they're sequential and I need those to be in parallel like in the example below:

[Link to example](https://codesandbox.io/s/objective-booth-8kb10?file=/src/App.tsx)

Is there any particular config am I missing? 

Thanks

**Comment by joshuaellis** [comment, marked as answer]
I don't believe this is a feature we have.
